pkg/api/router.go

```diff
@@ -182,6 +182,13 @@ func (s *server) setupRouting() {
 		})),
 	)
 
+	handle("/stamps/topup/{id}/{amount}", web.ChainHandlers(
```

Comment:
this should _not_ be in the api. all postage APIs have moved to the debugAPI

Reply:
Why do we still have the postage file in api package?